doc/man3/SSL_groupid_to_name.pod

```diff
@@ -0,0 +1,42 @@
+=pod
+
+=head1 NAME
+
+SSL_groupid_to_name - get name of group
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ const char *SSL_groupid_to_name(const SSL *ssl, int group_id);
+
+=head1 DESCRIPTION
+
+SSL_groupid_to_name() is used to retrieve the algorithm name
+associated with a given group_id and as registered via built-in
```

Comment:
`I<group_id>`
However it would be also better to say, what the group_id number actually is. That it is the TLS group ID.

Reply:
I don't think that is the correct semantics for this function. It should accept the nid, or the group id with the `TLSEXT_nid_unknown` bit set - but not the plain group id. That way it can be used to print the name on the results of functions like `SSL_get_shared_group`. For this reason I would call the function `SSL_group_to_name()` and not `SSL_groupid_to_name()`.

Reply:
Done.